src/ops/info.rs

```diff
@@ -128,6 +124,10 @@ fn try_list_owners(
     source_ids: RegistrySourceIds,
     package_name: &str,
 ) -> CargoResult<Option<Vec<String>>> {
+    // Only remote registries support listing owners.
+    if !source_ids.original.is_remote_registry() {
+        return Ok(None);
+    }
```

Comment:
We could check if the package is published and ask that registry about it.

However, considering `publish = true` is a default, we could end up with disclosures of private information.